manager/dispatcher/orphans_test.go

```diff
@@ -0,0 +1,84 @@
+package dispatcher
+
+import (
+	"runtime/debug"
+	"testing"
+	"time"
+
+	"github.com/docker/go-events"
+	"github.com/docker/swarmkit/api"
+	"github.com/docker/swarmkit/manager/state"
+	"github.com/docker/swarmkit/manager/state/store"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+)
+
+func TestOrphans(t *testing.T) {
+	s := store.NewMemoryStore(nil)
+	assert.NotNil(t, s)
+	defer s.Close()
+
+	downNodeCheckInterval = 100 * time.Millisecond
+	downNodeOrphanThreshold = time.Second
+	downNodeCheckMulti = (downNodeOrphanThreshold / downNodeCheckInterval)
```

Comment:
This is racy if any unit tests can run in parallel. The race detector complains about it.